falco/values.yaml

```diff
@@ -327,6 +327,78 @@ falcosidekick:
+    # -- The image repository to pull from.
+    repository: falcosecurity/falcoctl
+    #  -- Overrides the image tag whose default is the chart appVersion.
+    tag: "main"
+  artifact:
+    # -- Runs "falcoctl artifact install" command as an init container. It is used to install artfacts before
+    # Falco starts. It provides them to Falco by using an emptyDir volume.
+    install:
+      enabled: true
+      # -- Extra environment variables that will be pass onto falcoctl-artifact-install init container.
+      env: {}
+      # -- Arguments to pass to the falcoctl-artifact-install init container.
+      args: ["--verbose"]
+      # -- Resources requests and limits for the falcoctl-artifact-install init container.
+      resources: {}
+      # -- Security context for the falcoctl init container.
+      securityContext: {}
+    # -- Runs "falcoctl artifact follow" command as a sidecar container. It is used to automatically check for
+    # updates given a list of artifacts. If an update is found it downloads and installs it in a shared folder (emptyDir)
+    # that is accessible by Falco. Rulesfiles are automatically detected and loaded by Falco once they are installed in the
+    # correct folder by falcoctl. To prevent new versions of artifacts from breaking Falco, the tool checks if it is compatible
+    # with the running version of Falco before installing it.
+    follow:
+      enabled: true
+      # -- Extra environment variables that will be pass onto falcoctl-artifact-follow sidecar container.
+      env: {}
+      # -- Arguments to pass to the falcoctl-artifact-follow sidecar container.
+      args: ["--verbose"]
+      # -- Resources requests and limits for the falcoctl-artifact-follow sidecar container.
+      resources: {}
+      # -- Security context for the falcoctl-artifact-follow sidecar container.
+      securityContext: {}
+  # -- Configuration file of the falcoctl tool. It is saved in a configmap and mounted on the falcotl containers.
+  config:
+    # -- List of indexes that falcoctl downloads and uses to locate and download artiafcts. For more info see:
+    # https://github.com/falcosecurity/falcoctl/blob/main/proposals/20220916-rules-and-plugin-distribution.md#index-file-overview
+    indexes:
+    - name: falcosecurity
+      url: https://falcosecurity.github.io/falcoctl/index.yaml
+    # -- Configuration used by the artifact commands.
+    artifact:
+      install:
+        # -- List of artifacts to be installed by the falcoctl init container.
+        refs: [falco-rules:0]
+        # -- Directory where the rulesfiles are saved. The path is relative to the container, which in this case is an emptyDir
+        # mounted also by the Falco pod. If you change this value, please make sure you change the mount path of the emptyDir inside
+        # the falcoctl-artifact-install init container.
+        rulesfilesdir: /rulesfiles
+        # -- Same as the one above but for the artifacts.
+        pluginsdir: /plugins
```

Comment:
This is not completely clear to me.

How could the user `change the mount path of the emptyDir inside...` since those values are hard-coded in the chart :point_down: ?

https://github.com/falcosecurity/charts/pull/453/files#diff-701550ac61b2456be7b7355b821a218487fff642078f5d0aa22f1be38c725a75R260-R263

We may use those values to actually set the mount path inside the container OR disallow the user to change those values.
I prefer the first option (IIRC, we already to the same with some other option, maybe the gRPC socket path? :thinking:). However, I haven't thoughts if there're implications.